src/main/java/com/jwierzb/parkingspaces/security/JwtTokenFilter.java

```diff
@@ -0,0 +1,45 @@
+package com.jwierzb.parkingspaces.security;
+
+import java.io.IOException;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.jwierzb.parkingspaces.exception.CustomException;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.web.filter.GenericFilterBean;
+
+
+public class JwtTokenFilter extends GenericFilterBean {
```

Comment:
czy spring nie dostarcza bibliotek żeby to zrobić na wyższym poziomie abstrakcji?